docs/ssf/libraries.md

```diff
@@ -1,15 +1,76 @@
-# hg_sdf Functions
+# External Libraries
 
-In addition to the built in SSF functions, we have included the open-source [hg_sdf library by mercury](https://mercury.sexy/hg_sdf/) for your convienence. This library provides a very large toolset for creating raymarched scenes. To avoid namespace collision, we have prefixed all available functions with `_`. The following documentation is provided from the Mercury team. For more information, check out [their website](https://mercury.sexy/hg_sdf/).
+## Overview
 
-## Helper Functions and Macros
+In addition to the built in SSF library, Synesthesia includes some external libraries to help create your scenes. These libraries are not compiled by default -- you need to include the files in your `main.glsl` file to use them. 
 
-### `_sgn()`
+So far, there are two external libraries available in Synesthesia: [lygia](#lygia) and [hg_sdf](#hg_sdf).
+
+## Custom Library
+
+You can create your own GLSL library by adding a folder called `glsl` in your user data folder. Any `.glsl` files in this folder can be included in your shader. Here is the location of your user data folder:
+
+- Windows: `C:\Users\{username}\AppData\Local\Synesthesia\Data`
+- Mac: `~/Library/Application Support/com.gravitycurrent.synesthesia/Data`
+
+For example, if you create the file `glsl/lib.glsl` within this Data folder, you could add `#include "lib.glsl"` at the top of your shader and use any functions from the file.
+
+**NOTE:** if you plan to share a scene that uses custom library code, you'll need to copy paste the library code into your shader, since other users won't have these library files.
+
+## lygia
+
+[Lygia](https://github.com/patriciogonzalezvivo/lygia) is an open-source library by Patricio Gonzalez Vivo, creator of many incredible shader resources including [The Book of Shaders](https://thebookofshaders.com/). 
+
+The library includes a wide array of functions that touch nearly every aspect of shader programming. It is very granular -- each function has its own file. There are also some files that group multiple functions. For example:
+
+```js
+// include a single function
+#include "lygia/math/rotate3d.glsl"
+#include "lygia/space/displace.glsl"
+#include "lygia/color/tonemap.glsl"
+
+#include "lygia/math.glsl" 					// all math functions
+#include "lygia/sdf.glsl"  					// all sdf functions
+#include "lygia/color/blend.glsl"		// all blend functions
+```
+
+To learn more about the library and all its functions/files, check out the [github](https://github.com/patriciogonzalezvivo/lygia) or [website](https://lygia.xyz/). Here are links to documentation for each section of the library (copied from the website):
+
+<ul>
+	<li><a href="https://lygia.xyz/math"><code>math/</code></a>: general math functions and constants: <code>PI</code>, <code>SqrtLength()</code>, etc. </li>
```

Comment:
we should lock in what version of Lygia we're using and make a note of it here. we can also link to that version of their docs ie `https://lygia.xyz/v1.1.4/math`